explore-cardano/understanding-the-delegation-mechanism.md

```diff
@@ -1,5 +1,34 @@
 # Understanding the Delegation Mechanism
 
-As Cardano is a proof-of-stake system, owning ada not only allows you to buy goods or services, but also confers upon you the right and obligation to participate in the protocol and create blocks. 
+The stake delegation is a mechanism added to the Cardano proof of stake (PoS) protocol that allows the protocol to scale even in a setting where the set of stakeholders is highly fragmented.
 
-These two uses can be separated by the delegation mechanism, meaning someone who owns ada can keep the spending power, while delegating the power to participate in the protocol to someone else, a stake pool. It is important to note that funds can be spent normally at any time, regardless of how they are delegated.
+Anyone who owns ada can participate in the stake delegation process while retaining their spending power, while simultaneously delegating the power to participate in the protocol to a stake pool. Note that you can spend your ada normally at any time, regardless of how you delegated it. This mechanism will enable stakeholders to delegate their “voting rights” i.e., the right to participate in the slot leader election  process in each epoch.
+
+Stake delegation gives rise to “stake pools” that act in the similar way to mining pools in the Bitcoin protocol. Stake pool operators must be online in order to generate blocks when they are selected as slot leaders.
+
+## Stake delegation requirements
+
+Delegating stake requires posting two certificates to the chain: a staking key registration, and a delegation certificate. Posting certificates requires funds, so a user setting up their first wallet will need a bootstrapping mechanism. This mechanism relies on the possibility of base addresses using a staking key before posting the registration certificate for that key.
+Delegation scheme
+
+With the concept of delegation, any stakeholder can allow a stake pool to generate blocks for the Cardano network and then the rewards will be paid by the protocol for all participants including the fees for the stake pool operators. A stakeholder can transfer the right to generate blocks by creating a proxy signing key that allows the delegate to sign messages.
+
+In order to limit the delegate’s block generation power to a certain range of epochs and slots, the stakeholder can limit the proxy signing key’s valid message space to strings ending with a slot number in a specific range of values. This simple scheme is secure due to the verifiability and prevention of misuse properties of proxy signature schemes. This ensures that any stakeholder can verify that a proxy signing key was actually issued by a specific stakeholder to a specific delegate and that the delegate can only use these keys to sign messages inside the key’s valid message space, respectively.
```

Comment:
I'm not sure where the idea of a "proxy signing key" came from. A stake holder delegates to a particular pool ID, which is the hash of the operator's verification key. There is no slot range associated with this delegation choice, though a stake holder can always re-delegate.

Reply:
I concur.